pulp_2to3_migration/app/migrations/0014_auto_20200921_1012.py

```diff
@@ -0,0 +1,53 @@
+# Generated by Django 2.2.15 on 2020-09-21 10:12
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('pulp_2to3_migration', '0013_fix_file_remote_migration'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='pulp2rpm',
+            name='repodata',
+        ),
+        migrations.RemoveField(
+            model_name='pulp2srpm',
+            name='repodata',
+        ),
+        migrations.AddField(
+            model_name='pulp2rpm',
+            name='filelists_template_gz',
+            field=models.BinaryField(null=True),
+        ),
+        migrations.AddField(
+            model_name='pulp2rpm',
+            name='other_template_gz',
+            field=models.BinaryField(null=True),
+        ),
+        migrations.AddField(
+            model_name='pulp2rpm',
+            name='primary_template_gz',
+            field=models.BinaryField(null=True),
+        ),
+        migrations.AddField(
+            model_name='pulp2srpm',
+            name='filelists_template_gz',
+            field=models.BinaryField(null=True),
+        ),
+        migrations.AddField(
+            model_name='pulp2srpm',
+            name='other_template_gz',
+            field=models.BinaryField(null=True),
+        ),
+        migrations.AddField(
+            model_name='pulp2srpm',
+            name='primary_template_gz',
+            field=models.BinaryField(null=True),
+        ),
+    ]
+
+# TODO: data migration - for consistency and in case some content hasn't been migrated before the upgrade.
```

Comment:
This migration doesn't contain data migration.  Suitable for tests/experiments only.